### PR TITLE
Seeds

### PR DESCRIPTION
**Pull request by liambarstad**
LIST OF CHANGES:
- seeds.rb requires all files in db/seeds directory
- install fakergem, betterlorem, imagesuckr
- seeds for musical_instruments, goldfish, and electronics

THINGS I'M UNSURE ON:
- sometimes imagesuckr api call fails
- method get_url for imagesuckr makes query to google images api every time it retrieves one image, could be overwritten

RSPEC GREEN?
no tests written

*MENTION GROUP MEMBER TO REVIEW PR*
@mimilettd @jimszalew @vaidashi

### CODE REVIEW

**Review by jimszalew — DISMISSED** [review]
Is there going to be a specific command that runs all of these seed files? Will rake db:seed catch them all? If not I think it would be worth putting all seeding functions into the same seed file so that db:seed will catch it.

**Comment by jimszalew** [comment]
Nevermind, saw that seed file requires all the other seed files. I still think this is inefficient at best. Also, travis seems to want a test database for the code to pass the CI checks.

**Review by jimszalew — CHANGES_REQUESTED** [review]
All of these seed files will most likely not work. Master branch has an updated database and relationships, requiring an item to belong to a store. These will need to be updated.

**Review by jimszalew — DISMISSED** [review]
Looks good, I like the iterative approach to creating items. I think this should work with our current setup.

**Comment by liambarstad** [comment]
@mimilettd @jimszalew @vaidashi 
I don't know why these checks are passing all of a sudden, but it looks good to go

File: lib/tasks/reset.rake [hunk 1]
```diff
@@ -0,0 +1,25 @@
+Dir['./db/seeds/*.rb'].each {|file| require file }
```
Comment (jimszalew) [inline lib/tasks/reset.rake:1]: I'm really sorry, I know you explained this file in person to us but I don't remember what it does. Can you give me a quick description please? From what I've seen it looks like it runs all these other files that delete all of the items in a given category. When would this be used?

**Pull request closed without merging**